src/status_codes.toit

```diff
@@ -362,6 +362,23 @@ One of the requested extensions is not supported by the server.
 */
 STATUS_NOT_EXTENDED ::= 510
 
+/**
+Status codes on a close packet on the Websockets protocol
```

Comment:
```suggestion
Status codes for close packets on the WebSocket protocol.
```

Reply:
Annoying thing is, that the toitdoc only applies to the first one.